### PR TITLE
capstone: Build for the right architectures

### PR DESCRIPTION
**Pull request by ryandesign**
#### Description

capstone:

* Build for the right architectures.
* Disable silent rules.

###### Type(s)

- [X] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 10.13.6 17G65
Xcode 9.4.1 9F2000 

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [X] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [X] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: devel/capstone/Portfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ PortSystem          1.0
 PortGroup           github 1.0
 
 github.setup        aquynh capstone 3.0.5
+revision            1
 categories          devel
 platforms           darwin
 maintainers         gmail.com:aquynh
```
Comment (pmetzger) [inline devel/capstone/Portfile]: You don't know his github handle, do you?

Comment (ryandesign) [reply devel/capstone/Portfile]: Looks like it's @aquynh; I updated the Portfile with that.

Comment (pmetzger) [reply devel/capstone/Portfile]: Thanks!

**Comment by pmetzger** [comment]
Merged. Thanks, @ryandesign!

**Pull request merged**